lib/storage/metadata/mongoclient/LogConsumer.js

```diff
@@ -63,16 +57,39 @@ class ListRecordStream extends stream.Transform {
         }
 
         const dbName = itemObj.ns.split('.');
+        let entry;
+        if (itemObj.op === 'i') {
+            entry = {
+                type: 'put',
+                key: itemObj.o._id,
```

Comment:
We should check for the existence of `o` and `o2` properties before accessing their children. I know that the format is probably guaranteed by Mongodb but it's best if we check for the properties cleanly.

Reply:
done